ddl/ddl_worker.go

```diff
@@ -167,7 +178,7 @@ func (w *worker) Close() {
 		w.sessPool.Put(w.sess.Session())
 	}
 	w.wg.Wait()
-	logutil.Logger(w.logCtx).Info("DDL worker closed", zap.String("category", "ddl"), zap.Duration("take time", time.Since(startTime)))
+	logutil.Logger(w.logCtx).Info("DDL worker closed", zap.Duration("take time", time.Since(startTime)))
```

Comment:
Missing `w.jobLogger(job)` here?

Reply:
There is no job ref here because the stop method is not related to any job. I removed `zap.String("category", "ddl")` because the logger in `w.logCtx` contains this field already: https://github.com/pingcap/tidb/pull/46443/files#diff-dfc42c5764e7e4ff9122a1db728ff1cb0dee56e72dbccefdb211018ccd444c73R143

Reply:
Ok, but I see that your function(`jobLogger `) actually has a check to determine if the job is empty. So I'm asking.